documentation/modules/ROOT/pages/payloads.adoc

```diff
@@ -174,6 +174,100 @@ A Property `errorMessages` with a Collection Characteristic and effective data t
 }
 ----
 
+[[characteristics-payload-mappings]]
+== Specific payload mappings for Characteristics
+
+This section describes how data payloads look like for specific xref:characteristics.adoc[Characteristics].
+
+|===
+| Characteristic Name | Corresponding JSON data type
+| `samm-c:Either` | A JSON object with the left field and the value, which is one of the types proposed in `xref:characteristics.adoc#either-characteristic[Either]`.
+|===
+
+
+Example for `samm-c:Either`, a Property `speedProperty` has Characteristic and effective data type `samm-c:Either` and the possible values
+`ErrorMessage` (`xref:characteristics.adoc#single-entity-characteristic[Entity]` data type), which shows why speed can't be retrieved, or '60' (`xsd:integer`) would be serialized in the JSON payload as follows:
+
+Example:
+
+[source,turtle,subs="attributes+"]
+----
+include::example$aspect-with-either-characteristic.ttl[tags=aspect-with-either-characteristic]
+----
+
+Generated payload:
+
+[source,json]
+----
+{
+  "speedProperty" : {
+    "left" : {
+      "errorDescription" : "yedUsFwdkelQbxeTeQOvaScfqIOOmaa",
+      "errorCode" : "eOMtThyhVNLWUZNRcBaQKxI"
+    }
+  }
+}
+----
+
+or
+
+[source,json]
+----
+{
+  "speedProperty": {
+    "left": 60
+  }
+}
+----
+
+Another one example for Property `speedProperty` has Characteristic and effective data type `samm-c:Either`. Field `samm-c:left` with name 'Status' has type `samm-c:Either` and field `samm-c:right` with name 'Speed has type `xsd:integer`.
```

Comment:
Adjust this paragraph and examples to match the example from either-declaration.ttl and the above paragraph